scss/main.scss

```diff
@@ -2,13 +2,13 @@
 @import "mixin.scss";
 
 // mofo-bootstrap bits
-
-@import "../node_modules/mofo-bootstrap/src/scss/custom/colors";
+@import "./mofo-bootstrap/mofo-bootstrap";
```

Comment:
include this `mofo-bootstrap.scss` instead of the compiled `mofo-bootstrap.css` (see server.js)